server/registry/internal/storage/delete.go

```diff
@@ -303,8 +325,8 @@ func (c *Client) DeleteDeploymentRevision(ctx context.Context, name names.Deploy
 
 func (c *Client) DeleteArtifact(ctx context.Context, name names.Artifact) error {
 	for _, model := range []interface{}{
-		models.Blob{},
 		models.Artifact{},
+		models.Blob{},
```

Comment:
Was this reordering functionally necessary? I was tempted to label it ["drive by fix"](http://tillenius.me/blog/2010/05/03/problems-with-drive-by-fixes/) but I'm wondering if there's an ordering sensitivity that we should note.

Reply:
This is basically the same optimization: There's would be no point in calling the DB to delete blobs if the Artifact doesn't exist.

Reply:
Makes sens, thanks!